components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -61,5 +64,38 @@ class BrowserMenu internal constructor(
 
     companion object {
         private const val MENU_ELEVATION_DP = 8
+
+        /**
+         * Determines the orientation to be used for a menu based on the positioning of the [parent] in the layout.
+         */
+        fun determineMenuOrientation(parent: View): BrowserMenu.Orientation {
+            val params = parent.layoutParams
+            return if (params is CoordinatorLayout.LayoutParams) {
+                if (params.gravity and Gravity.BOTTOM == Gravity.BOTTOM) {
```

Comment:
This line is a bit confusing. 🤔  `Gravity.BOTTOM == Gravity.BOTTOM` Looks a bit strange, took me a bit to understand that `params.gravity` bit-field int and you're checking that `params.gravity` contains the value `BOTTOM`.

I was wonder if is there a way to enunciate better the intent.

Maybe something like:

```kotlin 

if (params.gravity in Gravity.BOTTOM) {
}
//Or
if (params.gravity.contains(Gravity.BOTTOM)) {
}
private operator fun Int.contains(mask: Int): Boolean {
    return (this and mask) == mask
}
```

Reply:
Would it be more obvious with `()`?

```Kotlin
if ((params.gravity and Gravity.BOTTOM) == Gravity.BOTTOM) {
...
```

Reply:
It helps 👍 !